README.md

```diff
@@ -1,173 +1,132 @@
-# Cloudinary Vue SDK
-
+Cloudinary Vue SDK
+=========================
 [![Build Status](https://travis-ci.org/cloudinary/cloudinary-vue.svg?branch=master)](https://travis-ci.org/cloudinary/cloudinary-vue)
+## About
+The Cloudinary Vue SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
```

Comment:
Did we have docs review the wording here?

Reply:
Yes